packages/icons/src/icon-collections/SAP-icons.json

```diff
@@ -1,654 +1,2059 @@
-    "sort-ascending": "M82 288h351l15 32H65zm-48 96h446l15 32H17zm366-160H113l16-32h255zm-47-96H161l17-32h158z",
-    "sort-descending": "M65 192h383l-15 32H82zM17 96h478l-15 32H34zm367 224H129l-16-32h287zm-47 96H177l-16-32h192z",
-    "sort": "M405.022 493.932q-9.951 9.951-22.888 9.951t-21.893-9.951l-101.504-100.51q-11.942-11.941 0-22.888 4.975-4.975 10.946-4.975t10.947 4.975l84.587 84.587v-317.45q0-15.922 15.922-15.922t15.922 15.923v319.44l86.577-86.578q10.947-10.946 22.889 0 10.946 10.947 0 22.889zM253.761 105.827q10.946 11.942 0 22.888-4.976 4.976-11.444 4.976t-11.444-4.976l-86.578-86.577v319.44q0 15.922-15.922 15.922t-15.922-15.922V44.128l-84.587 84.587q-4.976 4.976-10.947 4.976t-10.946-4.976q-11.942-10.946 0-22.888L107.475 5.317q8.956-9.95 21.893-9.95t22.889 9.95z",
-    "sorting-ranking": "M256.5 0l219 224h-439zm0 46l-144 146h287zm0 466l-220-224h439zm0-46l143-146h-287z",
-    "sound-loud": "M224.5 65q12 0 22 9t10 23v318q0 14-10 23t-22 9-20-7l-99-80q-11-7-20-7h-53q-13 0-19-5t-9-11q-4-7-4-16V191q0-9 4-16 3-6 9-11t19-5h53q9 0 20-7l99-80q10-7 20-7zm0 32l-99 80q-19 14-40 14h-53v130h53q21 0 40 14l99 80V97zm119 24q33 22 53 57t20 77q0 44-21.5 80t-56.5 57l-18-22v-5q29-17 46.5-46t17.5-64-17.5-64-46.5-46zm69-69q46 35 73 87.5t27 115.5q0 67-31 122.5t-83 89.5l-20-25q46-30 74-79t28-108q0-56-24.5-102.5T389.5 75z",
-    "sound-off": "M245 178l-23-24q1 0 2-1l140-82q8-7 20-7t22 9 10 23v252l-32-32V96l-2 1-2 2zm51 186l83 83q-4-1-7.5-2t-7.5-4l-140-82q-5-4-16-5.5t-16-1.5h-64q-13 0-19-5t-9-11q-4-7-4-16V192q0-3 1-7 1-3 2-6.5t4-7.5l25 25v124h64l18 2q8 2 16 3.5t15 6.5zm216 125l-23 23L0 23 23 0z",
-    "sound": "M384 64q13 0 22.5 9t9.5 23v320q0 14-9.5 23t-22.5 9q-10 0-20-7l-107-82q-4-3-15-5t-17-2h-97q-12 0-18.5-5t-9.5-11q-4-7-4-16V192q0-9 4-16 3-6 9.5-11t18.5-5h97q6 0 17-1.5t15-5.5l107-82q10-7 20-7zm0 32l-1 1-2 2-106 81q-14 9-31.5 10.5T225 192h-97v128h97q1 0 18.5 1.5T275 332l106 81 2 2 1 1V96z",
-    "source-code": "M51.052 242.926q-7.97 9.961 0 18.926L189.51 412.265q2.988 4.98 2.988 10.957 0 9.96-5.977 13.945-4.98 3.985-10.957 3.985-7.969 0-14.941-6.973L4.234 261.852q-8.965-8.965 0-18.926L160.624 65.62q6.972-6.973 14.941-6.973 16.934 0 16.934 17.93 0 6.973-2.988 10.957zm456.217 0q8.965 9.961 0 18.926L350.88 434.18q-6.972 6.973-14.941 6.973-5.977 0-10.957-3.985-5.977-3.984-5.977-13.945 0-6.973 3.984-10.957l138.46-150.413q5.976-8.965 0-18.926L322.988 87.533q-3.984-3.984-3.984-10.957 0-17.93 16.934-17.93 7.969 0 14.941 6.973z",
-    "split": "M475 411q5 5 5 12 0 6-5 11
```

Comment:
I think the idea of the file format was to save space, so you don't have objects for all of the items. Therefore the format is (currently):

"data": [..... all icons data"]
"accData": [.... acc info for the ones that have it...]

and if we follow the same format it would become:

"ltr": ["checkmark", "accept", .....]

just for the ones with LTR. 

So I would suggest one of the following:
1) Move accData to the icons with it, and we have only one key in the file (data)
2) Revert the old file format, and add a new field under "accData", called "ltr" and list the ltr-only icons there.
because currently it's a mix of both (ltr data is with the icon, but acc data is separate).

Both before and after the JSON file is 117KB gzipped, so that didn't change at all.
@pskelin Do you mind that we just move accData to the icon data?

Reply:
Option 1 has been implemented